VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioComponentDetails.cpp

```diff
@@ -41,6 +41,9 @@
 
 namespace
 {
+
+bool ReplacementDialogOpen = false;
```

Comment:
Some alarm bells are ringing here: 
1. Is there a way to open a view as a modal? (i.e. you cannot do anything until this window is closed again?)
2. Are we sure that the Unreal UI code is synchronous and that we never run into race conditions? otherwise we should check a lock or something similar. 
3. Is there a reason you use a variable in a local namespace over a static variable in a parent class of the replacement dialogues inheriting SReplacementDialogWidget?